husky_ur5_moveit_config/config/kinematics.yaml

```diff
@@ -1,5 +1,10 @@
+#ur5_arm:
+#  kinematics_solver: ur_kinematics/UR5KinematicsPlugin
+#  kinematics_solver_search_resolution: 0.005
+#  kinematics_solver_timeout: 0.005
+#  kinematics_solver_attempts: 3
 ur5_arm:
-  kinematics_solver: ur_kinematics/UR5KinematicsPlugin
+  kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
```

Comment:
if we enabled UR kinematics plugins, moveit outputs following errors, strangely it still able to move arms using InteractiveMarker (it uses kdl plugin for backup?), 
The ur5 moveit package also use kdl instead of ur one, may be we can ask @ipa-fxm for a reason.
https://github.com/ros-industrial/universal_robot/blob/indigo-devel/ur5_moveit_config/config/kinematics.yaml
```
[ERROR] [1453278228.152906667]: Kin chain provided in model doesn't contain standard UR joint 'shoulder_lift_joint'.
[ERROR] [1453278228.153010414]: Kinematics solver of type 'ur_kinematics/UR5KinematicsPlugin' could not be initialized for group 'ur5_arm'
[ERROR] [1453278228.153315138]: Kinematics solver could not be instantiated for joint group ur5_arm.
```